website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Podman Desktop 1.11 Release
+description: Podman Desktop 1.11 has been released!
+slug: podman-desktop-release-1.11
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.11/light-mode.png
```

Comment:
IMHO this should be banner.png, and a second copy it needs to go in website/static/img/blog/podman-desktop-release-1.11/ for this link to work.